requirements.in

```diff
@@ -0,0 +1,24 @@
+alembic
+arrow
+bleach
+cached-property
+docutils
+Flask==0.10.1
```

Comment:
I figured Flask is one of our biggest requirements. Not that it changes often, but I didn't think we'd want a new version slipping through.